.github/workflows/setup-kicad-library-utils.yml

```diff
@@ -0,0 +1,38 @@
+name: setup-kicad-library-utils
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v5.0.0
+
+      - name: Run action
+        uses: ./setup-kicad-library-utils
+
+      - name: Verify installation
+        shell: bash
+        run: |
+          # Check if the directory exists
+          ls -la /opt/kicad-library-utils
+
+          # Check if the symbolic links exist
+          ls -la /usr/local/bin/klc-footprint
+          ls -la /usr/local/bin/klc-symbol
+
+          # Test if the scripts are executable
+          klc-footprint --help
+          klc-symbol --help
```

Comment:
I don't think we need to check directories/links; just check that the commands can be invoked will do.